src/util.cpp

```diff
@@ -11,13 +13,10 @@ double sigmoid_diff(double width, double x){
 
 double getRampFactor(const double time, const double tstart, const double tstop, const double tramp){
 
-    double eps = 1e-4; // Cutoff for sigmoid ramp 
```

Comment:
I might need those later on at some point. Let's comment it out rather than removing. Same for the other removals within this file.